src/components/userGroups/AppPermissions.tsx

```diff
@@ -605,16 +605,18 @@ export default function AppPermissions({
                         Helm Apps
                     </NavLink>
                 </li>
-                <li className="tab-list__tab">
-                    <NavLink
-                        to={_getNavLinkUrl('jobs')}
-                        data-testid="jobs-permission-tab"
-                        className="tab-list__tab-link"
-                        activeClassName="active"
-                    >
-                        Jobs
-                    </NavLink>
-                </li>
+                {serverMode !== SERVER_MODE.EA_ONLY && (
+                    <li className="tab-list__tab">
+                        <NavLink
+                            to={_getNavLinkUrl('jobs')}
+                            data-testid="jobs-permission-tab"
+                            className="tab-list__tab-link"
+                            activeClassName="active"
+                        >
+                            Jobs
+                        </NavLink>
+                    </li>
+                )}
```

Comment:
We should remove the corresponding Route as well otherwise it remains accessible via URL

Reply:
Done